atomicapp/providers/lib/kubeshift/openshift.py

```diff
@@ -180,6 +180,8 @@ def _generate_kurl(self, obj, namespace, name=None, params=None):
                 url = self.k8s_api
             else:
                 url = urljoin(self.k8s_apis, "%s/" % api_version)
+        elif resource in self.oc_api_resources:
+            url = self.oc_api
```

Comment:
This does LGTM, but I'm just going to make sure this *still* works when creating k8s resources via OpenShift.